data/mods/Magiclysm/Spells/earthshaper.json

```diff
@@ -66,5 +66,91 @@
     "energy_increment": 75,
     "final_energy_cost": 1000,
     "difficulty": 5
+  },
+  {
+    "id": "eshaper_shardspray",
+    "type": "SPELL",
+    "name": "Shardspray",
+    "description": "This spell projects a wide spray of sharp metal shards, cutting into your foes and friends alike.",
+    "valid_targets": [ "hostile", "ground", "ally" ],
+    "effect": "cone_attack",
+    "spell_class": "EARTHSHAPER",
+    "energy_source": "MANA",
+    "difficulty": 3,
+    "base_casting_time": 100,
+    "base_energy_cost": 100,
+    "max_level": 20,
+    "min_damage": 10,
+    "max_damage": 60,
+    "damage_increment": 2,
+    "damage_type": "cut",
+    "min_aoe": 25,
+    "max_aoe": 60,
+    "aoe_increment": 5,
+    "min_range": 1,
+    "max_range": 10,
+    "range_increment": 0.5
+  },
+  {
+    "id": "eshaper_piercing_bolt",
+    "type": "SPELL",
+    "name": "Piercing Bolt",
+    "description": "This spell projects a piercing rod of conjured iron at those that dare oppose you.",
+    "valid_targets": [ "hostile" ],
+    "effect": "projectile_attack",
+    "spell_class": "EARTHSHAPER",
+    "energy_source": "MANA",
+    "base_casting_time": 100,
+    "base_energy_cost": 100,
+    "difficulty": 2,
+    "max_level": 20,
+    "min_damage": 20,
+    "max_damage": 50,
+    "damage_increment": 3,
+    "damage_type": "stab",
+    "min_range": 8,
+    "max_range": 26,
+    "range_increment": 1.5
+  },
+  {
+    "id": "eshaper_shardstorm",
+    "type": "SPELL",
+    "name": "Shardstorm",
+    "description": "Creates an omnidirectional spray of razor sharp metal shards all around you.",
+    "valid_targets": [ "hostile", "ally", "ground" ],
+    "min_damage": 20,
+    "max_damage": 60,
+    "damage_increment": 1.5,
+    "damage_type": "cut",
+    "min_aoe": 360,
```

Comment:
360 aoe? what?

Reply:
lol, I think that's just a typo/brainfart combination on my part. I was thinking of a 360 degree arc as though it was a cone spell, since that's what I want it to be. Though I guess effectively a 360 degree arc as a cone spell is the same as an aoe of whatever the range of the first spell is. Which is better to you?

Reply:
nah, leave it as a "target_attack" and input the radius as the aoe